### PR TITLE
Adds location of visitor's death

### PR DESCRIPTION
**Pull request by PowerSchill**
Closes #https://github.com/ldtteam/minecolonies-features/issues/141
Closes #
Closes #

# Changes proposed in this pull request:
-
-
-

Review please

### CODE REVIEW

**Comment by Raycoms** [comment]
Did you re-order them on purpose?

**Comment by PowerSchill** [comment]
No. IDEA "Reformat Code"

**Review by ravenbuilder934 — COMMENTED** [review]
Might make more sense if we switched it around so it said the location first then the cause, but that's just me.

File: src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java [hunk 1]
```diff
@@ -663,10 +663,18 @@ public void onDeath(DamageSource cause)
                     tavern.setNoVisitorTime(world.getRandom().nextInt(5000) + 30000);
                 }
 
+                final String deathLocation = "{".concat(String.valueOf(getCitizenData().getLastPosition().getX()))
```
Comment (Raycoms) [inline src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java]: I think we have a utility method somewhere for this.

Comment (PowerSchill) [reply src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java]: Any idea where to look

Comment (Raycoms) [reply src/main/java/com/minecolonies/coremod/entity/citizen/VisitorCitizen.java]: BlockPosUtil.getString()

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/ldtteam/minecolonies/pull-request/5645/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ldtteam/minecolonies)

**Review by someaddons — APPROVED** [review]

**Pull request merged**